pepper-apis/studybuilder-cli/studies/osteo/patches/osteo-pdfs.conf

```diff
@@ -673,5 +673,308 @@
         }
       ]
     },
+    {
+      "name": "osproject-release",
```

Comment:
Can you please add some notes on how these new pdf configuration gets loaded into the database ?

Reply:
Since this is OS2, should we update the name of this configration and/or the filename so that we can resolve the issue of colliding PDFs between OS1 and OS2 in DSM?  CC @ssettipalli

Reply:
> Can you please add some notes on how these new pdf configuration gets loaded into the database ?
> OsteoV2Updates which has sub tasks including OsteoMRFv2 is already applied to prod.
@ssettipalli Thanks for warning, I totally forgot that OsteoV2Updates is already on prod. Now it should be okay.

Reply:
@leonidrakitin :
just to be clear:
OsteoV2Updates is deployed to prod but issue reported in this ticket DDP-8721 exists and need a fix.